specify/specify7
# Add an Add and Delete tree ranks option

Fixes #598 

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions
- Open a tree 
- Click on the plus icon in the header, located between the pencil and the first rank (Planet | Earth | Root | ...) 
- Choose a parent rank 
- Click continue 
- Define the proprieties of your new rank 
- save 

- [x] verify that the new node has been added into the tree ranks 

- Open a tree 
- Click on the pencil icon in the header, located at the front of the headers, before the first rank (Planet | Earth | Root | ...) 
- Click on a pencil next to a tree rank 
- Click delete 
- Confirm deletion 

- [x] verify that the new node has been deleted from the tree ranks
